arelle/PackageManager.py

```diff
@@ -28,6 +29,21 @@
 
 EMPTYDICT = {}
 
+TAXONOMY_PACKAGE_FILE_NAMES = ('.taxonomyPackage.xml', 'catalog.xml') # pre-PWD packages
+
+reportPackageDirPattern = re.compile(r"^[^/]+/META-INF/reportPackage.json$|^[^/]+/reports/")
+
+reportPackageDocTypeExtensions = {
+    "https://xbrl.org/report-package/2023/xbri": (".xbri",),
```

Comment:
I know ESEF filing manual referenced the [CR](https://www.xbrl.org/Specification/report-package/CR-2023-05-03/report-package-CR-2023-05-03.html#sec-version-identifier).  Did you want to add those URIs too?